object/check.go

```diff
@@ -275,6 +276,17 @@ func CheckLdapUserPassword(user *User, password string, lang string) (string, er
 			return ldapServer.Id, fmt.Errorf(i18n.Translate(lang, "check:Multiple accounts with same uid, please check your ldap server"))
 		}
 
+		userDisabled = checkIsUserDisabled(searchResult.Entries[0].Attributes)
```

Comment:
Are we sure that the first item is always exist?

Reply:
```
                if len(searchResult.Entries) == 0 {
			conn.Close()
			continue
		}
		if len(searchResult.Entries) > 1 {
			conn.Close()
			return ldapServer.Id, fmt.Errorf(i18n.Translate(lang, "check:Multiple accounts with same uid, please check your ldap server"))
		}
```

above, here we check it out